stable/grc/templates/config-policy-clustermanagementaddon.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: config-policy-controller
+spec:
+  addOnMeta:
+    description: Distributes configured policies and monitors Kubernetes-based policies.
```

Comment:
This should be something like `Audits k8s resources and remediates violation based on configuration policies`